### PR TITLE
Issue/#37 Add to cart from PDP

### PR DESCRIPTION
**Pull request by nikola-mladenovic**

### CODE REVIEW

File: Sunrise/ViewModels/ProductViewModel.swift [hunk 1]
```diff
@@ -16,16 +18,40 @@ class ProductViewModel {
     let price = MutableProperty("")
     let oldPrice = MutableProperty("")
     let imageUrl = MutableProperty("")
+    let quantities = (1...9).map { String($0) }
+    let isLoading = MutableProperty(false)
+
+    // Dialogue texts
+    let addToCartSuccessTitle = NSLocalizedString("Product added to cart", comment: "Product added to cart")
+    let addToCartSuccessMessage = NSLocalizedString("You have successfully added product to your cart. Would you like to continue looking for more, or go to cart overview?", comment: "Product added to cart message")
```
Comment (cneijenhuis) [inline Sunrise/ViewModels/ProductViewModel.swift]: I think we can drop the first sentence, as the title already contains that info?

Comment (nikola-mladenovic) [inline Sunrise/ViewModels/ProductViewModel.swift]: Agreed 👍

**Comment by cneijenhuis** [comment]
I'd prefer it if after I click on `Cart Overview`, the navbar would show a back button to the PDP I came from.

**Comment by nikola-mladenovic** [comment]
Due to the way `UITabBarController`s and `UINavigationController`s work, we can quickly do one of these two implementations:
1. The way it is now - after tapping on _Cart Overview_, you switch the active tab to cart, and the cart screen is displayed. There is no back button, cause this screen belongs to a different tab, and it is not common to go back across tabs, but rather switch a tab, and then navigate back / forth within one tab only.
2. Perform `show` segue when tapping on _Cart Overview_ which would give us the back button, but the active tab won't change (meaning cart tab wouldn't be active, but the one we were at during PDP).

If we want a third approach that would be a bit more complicated from the routing perspective, but doable. WDYT @cneijenhuis ?

**Comment by cneijenhuis** [comment]
Okay, I've never worked with the TabBar so I didn't know that... then let's stick to the way it is now.
LGTM then 👍

**Pull request merged**